environments/production/postgresql.yml

```diff
@@ -19,7 +19,7 @@ dbs:
     host: hqdb3
     query_stats: True
   synclogs:
-    host: pgsynclog
+    host: pgsynclog1
```

Comment:
Won't this make processes start reading and writing to `pgsynclog1`? Sounds like we can do that until `pgsynclog1` is promoted to master

Reply:
It will be merge and deployed only after I promote it. There is a ticket for the same.